test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -9830,6 +9830,7 @@ SimpleTestPackageContext createTestPackage(string name, string version, List<Sim
                    pathContext.SolutionRoot,
                    NuGetFramework.Parse("netcoreapp2.0"));
                 projectA.Properties.Add("ManagePackageVersionsCentrally", "true");
+                projectA.Properties.Add("TransitiveDependencyPinningEnabled", "true");
```

Comment:
This class has access to `ProjectBuildProperties` so let's use the constants

```suggestion
                projectA.Properties.Add(ProjectBuildProperties.ManagePackageVersionsCentrally, "true");
                projectA.Properties.Add(ProjectBuildProperties.TransitiveDependencyPinningEnabled, "true");
```